nav2_amcl/include/nav2_amcl/pf/eig3.hpp

```diff
@@ -1,3 +1,23 @@
+/*
```

Comment:
I tracked down the nav stack to figure out who was the author. I was not able to find the exact user, because the accound who push the code 13 years ago is inactive now. So I just used the license from the other files and hope no one gets mad at us.

Reply:
Don't do that - just leave it blank. There are a ton of files in amcl that don't have headers and we really cant tell who they are this far in the past